GameData/RealismOverhaul/Engine_Configs/H1_Config.cfg

```diff
@@ -4,65 +4,69 @@
 //	Manufacturer: Rocketdyne
 //
 //	=================================================================================
-//	H-1 165K
+//	H-1 165K // Assuming this follows the A-B-C-D - H-1 A
```

Comment:
The C/D distinction was inboard/outboard (and thus differences in turbine exhaust and whether there was gimbal). I would guess (though I have not dug enough to know) that there was a similar distinction on early Saturn Is. This is relevant to the configs because some variants should have no gimbal (and thus be lighter).

Reply:
The original configs did not make the distinction, all 4 variants had the same (at least in mass, if they made a distinction in gimbal I did not change it.) I can create a distinction, but the changes would likely be arbitrary until I can find sources with data regarding that.

Reply:
Oh weird, maybe that's particular to ROEngines, rather than in the RO configs? I'll check tomorrow.

Reply:
The H-1A-B-C-D designations were based on what vehicle and position the H-1 was installed in (I believe Saturn 1 Inboard/outboard and Saturn 1B inboard/outboard), while the engine configs are based on the gradual uprating of the H-1 over the years (an H-1C/D on an early launch would have made 185klbf of thrust, while one on a later launch would have made 205klbf of thrust)